debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/OutboxEventRouterIT.java

```diff
@@ -23,12 +23,13 @@
 import org.apache.kafka.connect.header.Header;
 import org.apache.kafka.connect.header.Headers;
 import org.apache.kafka.connect.source.SourceRecord;
-import org.codehaus.jackson.JsonNode;
```

Comment:
That's interesting. It looks like we should exclue this (transitive) dependency, as we effectively have Jackson twice on the (test) classpath. Logged as https://issues.redhat.com/browse/DBZ-1943; could you take a look? Thanks!

Reply:
Ah; good catch.  Will do.